rialto_airflow/publish/publication.py

```diff
@@ -57,14 +61,12 @@ def export_publications(snapshot) -> bool:
         )
 
         with get_session(RIALTO_REPORTS_DB_NAME).begin() as insert_session:
-            insert_session.connection(
+            conn = insert_session.connection(
                 execution_options={"isolation_level": "SERIALIZABLE"}
             )
-            insert_session.connection().execute(
-                f"TRUNCATE {Publications.__tablename__}"
-            )
+            conn.execute(f"TRUNCATE {Publications.__tablename__}")
 
-            for row in select_session.execute(stmt):
+            for count, row in enumerate(select_session.execute(stmt), start=1):
```

Comment:
This looks convoluted in the diff but is basically just using the result of calling connection() to then execute a query.